riscv.h

```diff
@@ -117,6 +131,9 @@ void rv_set_reg(struct riscv_t *, uint32_t reg, riscv_word_t in);
 /* get a register of the RISC-V emulator */
 riscv_word_t rv_get_reg(struct riscv_t *, uint32_t reg);
 
+/* get a byte from the memory of the RISC-V emulator */
+uint8_t rv_get_mem(struct riscv_t *, uint32_t reg);
```

Comment:
Is it worthy to provide this helper function?

Reply:
Those new interfaces in `riscv.h` are added to access the `struct riscv_t` originally. I change to include `riscv_private.h` in `gdbstub.c` directly to overcome this problem.